submarine-sdk/pysubmarine/submarine/ml/parameters.py

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+LIBSVM = "libsvm"
+
+defaultParameters = {
+  "output": {
+    "save_model_dir": "./experiment",
+    "metric": "auc"
+  },
+  "training": {
+    "batch_size": 512,
+    "field_size": 39,
+    "num_epochs": 3,
+    "feature_size": 117581,
+    "embedding_size": 256,
+    "learning_rate": 0.0005,
+    "batch_norm_decay": 0.9,
+    "l2_reg": 0.0001,
+    "deep_layers": "400,400,400",
+    "dropout": "0.3,0.3,0.3",
+    "batch_norm": "false",
+    "optimizer": "adam",
+    "log_steps": 10,
+    "num_threads": 4,
+    "num_gpu": 0,
+    "seed": 77,
+    "mode": "local"
+  }
+}
```

Comment:
How common this parameter to cover other models? If it is common enough, we can leave it either here or move it to model. If it is more specific to deepFM model, we shoud move it to model/deep_fm.

Reply:
They cover all other models, except `deep_layers`

- In my opinion, the advantage of leaving all parameters in JSON is that users can easily tune their parameters in one file, and the JSON spec will be refined in the future.
- Actually, if we specify parameters that not include in Deepfm, it will not affect training.
The model will only get the parameters they required.
- I will add a doc for this JSON spec so that users can know what kind of parameters that need to write at the beginning